### PR TITLE
editoast: return list of missing locations in pathfinding

### LINKED ISSUE #8636: Editoast: being able to identify unrecognized PR in the infrastructure
If one or more PRs are missing in the infrastructure during pathfinding, an error will be sent to the front-end containing the list of unrecognized PRs.

### PR DESCRIPTION
**Pull request by hamz2a**
closes https://github.com/OpenRailAssociation/osrd/issues/8636

### CODE REVIEW

**Review by SharglutDev — APPROVED** [review]
Lgtm for front part, not tested (we don't display these errors for now)

**Review by flomonster — APPROVED** [review]
Thanks! LGTM

File: editoast/src/views/path/pathfinding.rs [hunk 1]
```diff
@@ -332,3 +332,111 @@ fn path_input_hash(infra: i64, infra_version: &String, path_input: &PathfindingI
...
+            .response(StatusCode::OK)
+            .body(
+                r#"{
+                "blocks":[],
+                "routes": [],
+                "track_section_ranges": [],
+                "path_item_positions": [],
+                "length": 0,
+                "status": "success"
+                }"#,
```
Comment (leovalais) [inline editoast/src/views/path/pathfinding.rs]: You can also use `serde_json::json!` here.

**Review by leovalais — APPROVED** [review]
Great work! (Not tested.)

**Review by younesschrifi — APPROVED** [review]

File: editoast/src/views/path/pathfinding.rs (hunk 1 above)
Comment (hamz2a) [reply editoast/src/views/path/pathfinding.rs]: Done.

**Review by leovalais — APPROVED** [review]
Thanks!

**Pull request merged**